lib/one_model.ex

```diff
@@ -852,10 +891,11 @@ defmodule OneModel do
   @doc """
   Safe conversion of string to integer.
 
-  Allows nil, integers, and stings to be converted with the following behavior:
+  Allows nil, integers, and strings to be converted with the following behavior:
```

Comment:
I believe this should be spelled `behaviour` to be consistent with the Elixir keyword. :)